src/main.py

```diff
@@ -14,29 +28,55 @@ def is_controller_connected():
 
 
 """
-Script to determine leader and follower roles.
+This function calls the ./bt_toggle script with the specified
+command and address.
+"""
+def bt_toggle(command, address):
+    subprocess.call(['./bt_toggle.sh', command, address])
 
-Takes input from the camera to determine if the current vehicle
-should lead or follow based on whether or not there exists an ARTag
-in front of the vehicle.
 
-If an ARTag exists, the vehicle will become a follower vehicle.
-If an ARTag does not exist, the vehicle will become the leader vehicle.
-"""
 def main():
     tag = TagRecognition()
+    bl = Bluetoothctl()
+    BT_ADDRESS = '5C:BA:37:26:6D:9A'
 
-    # If an ARTag is detected, there is a vehicle ahead of this vehicle so it will follow.
-    # If an ARTag is not detected, no vehicle is present so this vehicle will be a leader.
+    # If an ARTag is detected, the vehicle will disconnect from a Bluetooth
+    # controller. If an ARTag is not detected, the vehicle will connect to
+    # a Bluetooth controller.
     if tag.detect():
-        tag = None
-        follower = Follower()
-        while True:
-            follower.follow()
+        while is_controller_connected():
```

Comment:
Comment applies to all the Bluetooth stuff:

not quite how I was thinking we'd use this. Take a look at my branch and see how I'm using it and validating that events actually happened.

Reply:
But again, I'm not sure it should be in this PR because we need to fix it before we put role swapping in.

Reply:
Ok, I'll leave the connect and disconnect functions in there just as place holders until we decide how we are going to use the python wrapper